<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Car Controller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_5">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame
{
	
	background-color: rgb(176, 176, 176);
	
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="connectBt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}

QPushButton:hover
{
	border: 2px solid rgb(255, 255, 255);
	
	border-radius: 10px;
}

QPushButton:pressed
{
	border: 2px solid rgb(255, 0, 0);
	
	border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DisconnectBt">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}

QPushButton:hover
{
	border: 2px solid rgb(255, 255, 255);
	
	border-radius: 10px;
}

QPushButton:pressed
{
	border: 2px solid rgb(255, 0, 0);
	
	border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ControlButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}

QPushButton:hover
{
	border: 2px solid rgb(255, 255, 255);
	
	border-radius: 10px;
}

QPushButton:pressed
{
	border: 2px solid rgb(255, 0, 0);
	
	border-radius: 10px;
}</string>
         </property>
         <property name="text">
          <string>Control</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="connectedButton">
         <property name="text">
          <string>Connected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disconnectedButton">
         <property name="text">
          <string>Disconnected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">QStackedWidget
{
	
	
	background-color: rgb(250, 255, 199);
}</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="controlPage">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Joypad</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QWidget" name="widget_5" native="true">
                   <property name="styleSheet">
                    <string notr="true">Qwidget
{
	border: 2px solid rgb(255, 255, 255);
}</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="leftButton">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton
{
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}

QPushButton:hover
{
	border: 2px solid rgb(85, 170, 255);
	
	border-radius: 10px;
}

QPushButton:pressed
{
	border: 2px solid rgb(255, 0, 0);
	
	border-radius: 10px;
}</string>
                      </property>
                      <property name="text">
                       <string>L</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QVBoxLayout" name="verticalLayout_4">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="forwardButton">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton
{
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}

QPushButton:hover
{
	border: 2px solid rgb(85, 170, 255);
	
	border-radius: 10px;
}

QPushButton:pressed
{
	border: 2px solid rgb(255, 0, 0);
	
	border-radius: 10px;
}</string>
                      </property>
                      <property name="text">
                       <string>F</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="backButton">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton
{
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}

QPushButton:hover
{
	border: 2px solid rgb(85, 170, 255);
	
	border-radius: 10px;
}

QPushButton:pressed
{
	border: 2px solid rgb(255, 0, 0);
	
	border-radius: 10px;
}</string>
                      </property>
                      <property name="text">
                       <string>B</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QWidget" name="widget_7" native="true">
                   <layout class="QHBoxLayout" name="horizontalLayout_4">
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <item>
                     <widget class="QPushButton" name="rightButton">
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>25</height>
                       </size>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">QPushButton
{
	border: 2px solid rgb(68, 68, 68);
	
	border-radius: 10px;
}

QPushButton:hover
{
	border: 2px solid rgb(85, 170, 255);
	
	border-radius: 10px;
}

QPushButton:pressed
{
	border: 2px solid rgb(255, 0, 0);
	
	border-radius: 10px;
}</string>
                      </property>
                      <property name="text">
                       <string>R</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="widget_8" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Normal Speed</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_9" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>15</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Turn Speed</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="BtListPage">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="BtListWidget">
             <property name="styleSheet">
              <string notr="true"> QListWidget {
        background: #ffffff;
        border: 1px solid #ddd;
    }
    QListWidget::item {
        padding: 8px;
        border-bottom: 1px solid #eee;
    }
    QListWidget::item:hover {
        background-color: #f5f5f5;
    }
    QListWidget::item:selected {
        background-color: #4caf50;
        color: white;
    }</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
